libraries/apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/api/NormalizedCache.kt

```diff
@@ -146,5 +147,12 @@ abstract class NormalizedCache : ReadOnlyNormalizedCache {
 
     private val specialChars = "()^$.*?+{}"
   }
+
+  /**
+   * Closes resources associated with the cache if any.
+   * This function must not call `nextCache.close()`, this is done by the caller.
```

Comment:
It's very easy to forget calling `nextCache.close()` so I went with this but no strong opinion there.